styles.css

```diff
@@ -0,0 +1,189 @@
+:root {
+	--dark-blue: #0a415c;
+	--my-white: #cfd2d4;
+}
+
+html {
+	font-family: "Trebuchet MS", sans-serif;
+	font-size: 10px;
+}
+
+body {
+	background-color: var(--dark-blue);
+	color: var(--my-white);
+}
+
+h1 {
+	position: absolute;
+	top: 50%;
+	left:50%;
+	transform: translate(-50%, -50%);
+	font-size: 4rem;
+	padding: 5px 100px;
+	text-transform: uppercase;
+	white-space: nowrap;
+	border: 2px solid;
+}
+
+h2, h3 {
+	border-bottom: 1px solid;
+	text-transform: uppercase;
+	white-space: nowrap;
+	}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
+p {
+	font-size: 2rem;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;	
+	align-items: center;
+}
+
+.nav-bar {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	gap: 20px;
+	font-size: 2rem;
+	padding: 20px;
+}
+
+.nav-bar a:link, .nav-bar a:visited {
+	color: var(--my-white);
+}
+.nav-bar a:hover {
+	color: #048230;
+}
+
+
+.cv-download-button {
+	padding: 5px 20px;
+	border: 2px solid #048230;
+	background-color: var(--dark-blue);
+	color: #048230;
+	font-size: 1.6rem;
+	cursor: pointer;
+	border-radius: 20px;
+	margin: 0 20px;
+}
+
+
+.cv-download-button:hover {
+	background-color: #048230;
+	color: var(--my-white);
+}
+
+header {
+	position: relative;
+}
+
+
+
+main {
+	margin-top: 20px;
+	display: grid;
+	grid-template-columns: 2fr 1fr;
+	max-width: 2000px;
+	padding: 20px;
+}
+
+header img {
+	width: 100%;
+}
+
+.portfolio {
+	border: 1px solid;
+	margin: 20px;
+	padding: 20px;
+}
+
+.portfolio-card-container{
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-evenly;
+}
+
+.portfolio-card {
+	border: 1px solid;
+	padding: 20px;
+	margin: 20px 0px;
+	background-color: var(--my-white);
+	color: var(--dark-blue);
+	width: 300px;
+}
+
+.portfolio-card img{
+	height: 200px;
+}
+
+.about {
+	display: flex;
+	flex-flow: column nowrap;
+	border: 1px solid;
+	margin: 20px;
+	padding: 20px;
+	min-width: 300px;
```

Comment:
this needs to be removed - it causes horizontal scroll.

but also - the image then becomes huge. then the image needs `width: 100%` so it isn't huge

and then everything should be fine and no horizontal scroll should happen